media-libs/opencv/opencv343-3.4.3.recipe

```diff
@@ -0,0 +1,285 @@
+SUMMARY="Open Source Computer Vision Library"
+DESCRIPTION="A collection of algorithms and sample code for various \
+computer vision problems."
+HOMEPAGE="https://opencv.org/"
+COPYRIGHT="2000-2016, Intel Corporation
+	2009-2011, Willow Garage Inc.
+	2009-2016, NVIDIA Corporation
+	2010-2013, Advanced Micro Devices, Inc.
+	2015-2016, OpenCV Foundation
+	2015-2016, Itseez Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/opencv/opencv/archive/$portVersion.tar.gz"
+SOURCE_FILENAME="opencv-$portVersion.tar.gz"
+CHECKSUM_SHA256="4eef85759d5450b183459ff216b4c0fa43e87a4f6aa92c8af649f89336f002ec"
+SOURCE_DIR="opencv-$portVersion"
+
+ARCHITECTURES="?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Comment:
The `SONAME` version for opencv [is actually `3.4`][0], so

```diff
-libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
```

[0]: https://abi-laboratory.pro/?view=timeline&l=opencv

Reply:
You would really need to bump everything with this change :P, but since `SONAME` is kept, you might not need to check every package after bump...

Reply:
still not resolved